File: MPI/mpicheck/tests/functions/3.0/MPI_Status_f2f08.f90

        
        program main
        use mpi
        
        INTEGER var_0(10)
       TYPE(MPI_Status) var_1
       INTEGER var_2
        call mpi_status_f2f08(var_0, var_1, var_2)
       call pmpi_status_f2f08(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/3.0/MPI_Status_f2f08.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const MPI_Fint *var_0;
    MPI_F08_status *var_1;
    int ret;
    /* calls */
    ret = MPI_Status_f2f08(var_0, var_1);
    ret = PMPI_Status_f2f08(var_0, var_1);
    return 0;
}